Update packages, add Eslint AirBNB config

nienkedekker opened the pull request:
What it says on the tin :)

I also ran Prettier, so only `package.json` is important to check out - the other changes are formatting changes.

lazeratops reviewed (approved):
lgtm with one question re pre-commit hook.


This may be out of scope for this particular PR, but at some point it might be good to implement the Airbnb eslint style ruleset for this demo, to be consistent with our new boilerplate linter that will have airbnb across the board

nienkedekker commented:
> lgtm with one question re pre-commit hook.
> 
> This may be out of scope for this particular PR, but at some point it might be good to implement the Airbnb eslint style ruleset for this demo, to be consistent with our new boilerplate linter that will have airbnb across the board

1. I removed Husky for now - I want to test it more, and then create a separate PR for it!
2. I added the Airbnb Eslint config. I did disable some ESLint rules in eslintrc.js to avoid errors - fixing them now is outside of the scope of this PR I think.

lazeratops reviewed (approved):
Great!! 🚢
